### PR TITLE
Netsuite: add config so the user will be able to add typesToSkip

### PR DESCRIPTION
**Pull request by omrilit**

### CODE REVIEW

File: packages/netsuite-adapter/src/adapter.ts [hunk 1]
```diff
@@ -64,9 +72,11 @@ export default class NetsuiteAdapter implements AdapterOperations {
       SAVED_SEARCH, // Due to https://github.com/oracle/netsuite-suitecloud-sdk/issues/127 we receive changes each fetch
     ],
     getElemIdFunc,
+    config,
   }: NetsuiteAdapterParams) {
     this.client = client
     this.typesToSkip = typesToSkip
+    this.typesToSkip = typesToSkip.concat(makeArray(config[TYPES_TO_SKIP]))
```
Comment (adamro) [inline packages/netsuite-adapter/src/adapter.ts:79]: is line 78 needed?

File: packages/netsuite-adapter/src/adapter_creator.ts [hunk 2]
```diff
@@ -34,6 +39,26 @@ const credentialsType = new ObjectType({
   annotations: {},
 })
 
+const configType = new ObjectType({
```
Comment (adamro) [inline packages/netsuite-adapter/src/adapter_creator.ts:42]: you don't have a `types.ts` file to put it there?

File: packages/netsuite-adapter/src/adapter.ts (hunk 1 above)
Comment (omrilit) [reply packages/netsuite-adapter/src/adapter.ts:79]: Of course not... just a leftover.

File: packages/netsuite-adapter/src/adapter_creator.ts (hunk 2 above)
Comment (omrilit) [reply packages/netsuite-adapter/src/adapter_creator.ts:42]: I do have such, but it's an auto-generated file.

**Pull request merged**